config.gradle

```diff
@@ -11,40 +11,40 @@
  * and fluentui_drawer' current version is 0.0.2, both fluentui_listitem and
  * fluentui_drawer and FluentUI should increment their respective version ids
   */
-project.ext.fluentui_calendar_versionid = '0.0.29'
-project.ext.fluentui_controls_versionid = '0.1.23'
-project.ext.fluentui_core_versionid = '0.1.34'
-project.ext.fluentui_listitem_versionid = '0.1.21'
-project.ext.fluentui_tablayout_versionid = '0.1.16'
-project.ext.fluentui_drawer_versionid = '0.1.31'
-project.ext.fluentui_ccb_versionid = '0.1.15'
-project.ext.fluentui_others_versionid = '0.1.8'
-project.ext.fluentui_transients_versionid = '0.0.35'
-project.ext.fluentui_topappbars_versionid = '0.1.16'
-project.ext.fluentui_menus_versionid = '0.1.7'
-project.ext.fluentui_peoplepicker_versionid = '0.1.6'
-project.ext.fluentui_persona_versionid = '0.1.20'
-project.ext.fluentui_progress_versionid = '0.1.10'
-project.ext.fluentui_icons_versionid = '0.1.8'
-project.ext.fluentui_notification_versionid = '0.1.15'
-project.ext.FluentUI_versionid = '0.1.47'
-project.ext.fluentui_calendar_version_code = 29
-project.ext.fluentui_controls_version_code = 24
-project.ext.fluentui_core_version_code = 57
-project.ext.fluentui_listitem_version_code = 46
-project.ext.fluentui_tablayout_version_code = 41
-project.ext.fluentui_drawer_version_code = 55
-project.ext.fluentui_ccb_version_code = 39
-project.ext.fluentui_others_version_code = 31
-project.ext.fluentui_transients_version_code = 35
-project.ext.fluentui_topappbars_version_code = 42
-project.ext.fluentui_menus_version_code = 35
-project.ext.fluentui_peoplepicker_version_code = 36
-project.ext.fluentui_persona_version_code = 45
-project.ext.fluentui_progress_version_code = 35
-project.ext.fluentui_icons_version_code = 9
-project.ext.fluentui_notification_version_code = 16
-project.ext.FluentUI_version_code = 82
+project.ext.fluentui_calendar_versionid = '0.1.0'
```

Comment:
Is there a reason for it to not be 0.0.30?
and same in fluentui_transients_versionid?

Reply:
we wanted to have all the artifacts to follow 0.1.x convention.